storage/innobase/buf/buf0rea.cc

```diff
@@ -788,6 +795,10 @@ buf_read_ahead_linear(
 			}
 		}
 	}
+#if defined(LINUX_NATIVE_AIO)
+	/* Tell aio to submit all buffered requests. */
```

Comment:
Useless comment (retells the called function name)

Reply:
removed